mmv1/third_party/terraform/tests/data_source_dns_managed_zone_test.go.erb

```diff
@@ -15,24 +15,50 @@ func TestAccDataSourceDnsManagedZone_basic(t *testing.T) {
 
 	vcrTest(t, resource.TestCase{
 		PreCheck:     func() { testAccPreCheck(t) },
-		Providers:    testAccProviders,
-		CheckDestroy: testAccCheckDNSManagedZoneDestroyProducer(t),
+		CheckDestroy: testAccCheckDNSManagedZoneDestroyProducerFramework(t),
 		Steps: []resource.TestStep{
 			{
+				ExternalProviders: providerVersion450(),
+				Config:            testAccDataSourceDnsManagedZone_basic(randString(t, 10)),
+				Check: checkDataSourceStateMatchesResourceStateWithIgnores(
+					"data.google_dns_managed_zone.qa",
+					"google_dns_managed_zone.foo",
+					map[string]struct{}{
+						"dnssec_config.#":                       {},
+						"private_visibility_config.#":           {},
+						"peering_config.#":                      {},
+						"forwarding_config.#":                   {},
+						"force_destroy":                         {},
+						"labels.#": 				                     {},
+						"creation_time": 			                   {},
+						"cloud_logging_config.#":                {},
+						"cloud_logging_config.0.%":              {},
+						"cloud_logging_config.0.enable_logging": {},
+<% unless version == "ga" -%>
+						"reverse_lookup":                        {},
+<% end -%>
+					},
+				),
+			},
+			{
+				ProtoV5ProviderFactories: protoV5ProviderFactories(t),
 				Config: testAccDataSourceDnsManagedZone_basic(randString(t, 10)),
 				Check: checkDataSourceStateMatchesResourceStateWithIgnores(
 					"data.google_dns_managed_zone.qa",
 					"google_dns_managed_zone.foo",
 					map[string]struct{}{
-						"dnssec_config.#":             {},
-						"private_visibility_config.#": {},
-						"peering_config.#":            {},
-						"forwarding_config.#":         {},
-						"force_destroy":               {},
-						"labels.#": 				   {},
-						"creation_time": 			   {},
+						"dnssec_config.#":                       {},
+						"private_visibility_config.#":           {},
+						"peering_config.#":                      {},
+						"forwarding_config.#":                   {},
+						"force_destroy":                         {},
+						"labels.#": 				                     {},
```

Comment:
I'm a bit suspicious about this test as there was the issue before where it turned out labels.# wasn't being used in the test using v4.50.0 (there wasn't a a labels attribute to be ignored in that test) and that line needed to be changed to labels.% to work with the muxed provider (as there was a labels attribute and it had a % not a #). See here : https://github.com/hashicorp/magic-modules/blob/06984badd83acfc915b9e82331bf19156dd7e00b/mmv1/third_party/terraform/tests/data_source_dns_managed_zone_test.go.erb

I don't understand how the vcr tests have passed on this pr given that issue... 🤷‍♀️